examples/storage/simpletest/README.md

```diff
@@ -0,0 +1,27 @@
+Hello, World!
+-----------------
+
+This is the simplest way to test persistent volumes.
+
+
+This example will serve an http response of "Hello Openshift!" to [http://localhost:6061](http://localhost:6061).  To create the pod run:
```

Comment:
Why openshift instructions? If it's really necessary, there needs to be a pointer to instructions about how to run openshift.